pyproject.toml

```diff
@@ -265,6 +265,9 @@ unfixable = [
     "E711"  # NoneComparison. Hard to fix b/c numpy has it's own None.
 ]
 
+[tool.flynt]
+exclude= ["astropy/extern"]
```

Comment:
Should we exclude generated files (`*lextab`, etc) as well? Clearly hasn't been an issue, so perhaps not worth it, but it may become so?

Reply:
Unfortunately, `flynt` does not accept wildcards there. It just matches the prefix on the relative path from the root.

Reply:
I guess hard-coding the paths is good for now. There are only 2 or 3 lextab and parsetab files.

Reply:
Done